compiler/src/main/java/org/qbicc/graph/ValueVisitor.java

```diff
@@ -384,10 +384,6 @@ default R visit(T param, ZeroInitializerLiteral node) {
         return visitUnknown(param, node);
     }
 
-    default R visit(T param, Deref node) {
-        return visitUnknown(param, node);
-    }
-
```

Comment:
Following up to the above note, the visitor and copy methods should be retained.